packages/schematics/angular/migrations/use-application-builder/migration.ts

```diff
@@ -178,6 +178,11 @@ function updateProjects(tree: Tree, context: SchematicContext) {
         continue;
       }
 
+      if (buildTarget.builder.startsWith('@angular/build')) {
```

Comment:
The check in https://github.com/Deku-nattsu/angular-cli/blob/9d7db9ea1b0837f282b4c0effe157170b0d6eca7/packages/schematics/angular/migrations/use-application-builder/migration.ts#L177C51-L177C71 should be updated to 

```
if (!buildTarget || buildTarget.builder === Builders.Application 
|| buildTarget.builder === Builders.BuildApplication) {
```